libraries/botbuilder-core/src/turnContext.ts

```diff
@@ -764,14 +765,25 @@ export class TurnContext {
      * Gets the locale stored in the turnState.
      */
     public get locale(): string {
```

Comment:
Update the turn type to include `| undefined`.

Reply:
fixed